pkg/operator/operator.go

```diff
@@ -176,7 +176,14 @@ func (optr *Operator) sync(key string) error {
 		glog.Errorf("Failed getting operator config: %v", err)
 		return err
 	}
-
+	if operatorConfig.Controllers.Provider == "None" {
+		glog.V(1).Info("`None` provider specified, reporting available")
+		if err := optr.statusAvailable(); err != nil {
+			glog.Errorf("Error syncing ClusterOperatorStatus: %v", err)
+			return fmt.Errorf("error syncing ClusterOperatorStatus: %v", err)
```

Comment:
@michaelgugino should we `optr.statusFailing` here?

Reply:
I don't think so.  If we fail to set statusAvailable it's most likely due to temporary network outage / api unavailable, so we want to retry I think.

Generally, if provider is None type, I think we want to do a no-op and report available.